HDDM.egg-info/PKG-INFO

```diff
@@ -0,0 +1,17 @@
+Metadata-Version: 1.1
```

Comment:
This whole dir shouldn't exist.

Reply:
gone. Not sure why this was still there since it had been included in .gitignore already